tests/simulator_avx_test.cc

```diff
@@ -16,45 +16,58 @@
 
 #include "gtest/gtest.h"
 
-#include "../lib/formux.h"
+#include "../lib/parfor.h"
+#include "../lib/seqfor.h"
```

Comment:
From the Kokoro failures, I would guess that these need to be in the `ifdef` block, similar to `formux.h`.